doc/lightning-fundchannel_cancel.7.md

```diff
@@ -11,10 +11,18 @@ SYNOPSIS
 DESCRIPTION
 -----------
 
-`fundchannel_cancel` is a lower level RPC command. It allows a user to
-cancel an initiated channel establishment with a connected peer.
+`fundchannel_cancel` is a lower level RPC command. It allows channel funder
+to cancel a channel before funding broadcast with a connected peer.
 
-*id* is the node id of the remote peer with which to cancel the
+*id* is the node id of the remote peer with which to cancel.
+
+Note that the funding transaction MUST NOT be broadcast before
+`fundechannel_cancel`. Broadcasting transaction before `fundechannel_cancel`
+can lead to unrecoverable loss of funds.
+
+If `fundechannel_cancel` is called after `fundchannel_complete`, the remote
```

Comment:
Extraneous `e` in `fundechannel_cancel`.

Reply:
Corrected.